.changeset/shiny-geckos-hang.md

```diff
@@ -0,0 +1,13 @@
+---
+"@sumup-oss/circuit-ui": major
+---
+
+Marked the `Calendar` component as stable. Update the related imports:
+
+```diff
+// Before
+- import { Calendar, CalendarProps, PlainDateRange } from '@sumup-oss/circuit-ui/experimental';
+// After
++ import { Calendar, CalendarProps, PlainDateRange } from '@sumup-oss/circuit-ui';
```

Comment:
The "Before" and "After" comments are already implied by the `diff` syntax:

```diff
- import { Calendar, CalendarProps, PlainDateRange } from '@sumup-oss/circuit-ui/experimental';
+ import { Calendar, CalendarProps, PlainDateRange } from '@sumup-oss/circuit-ui';
```

Personally, I'd remove the comments but I leave the decision up to you :)